resources/services/lightsail/buckets.go

```diff
@@ -0,0 +1,259 @@
+package lightsail
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/lightsail"
+	"github.com/aws/aws-sdk-go-v2/service/lightsail/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource buckets --config gen.hcl --output .
+func Buckets() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_lightsail_buckets",
+		Description:  "Describes an Amazon Lightsail bucket",
+		Resolver:     fetchLightsailBuckets,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("lightsail"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
```

Comment:
Are there regions for Lightsail buckets?

Reply:
@bbernays yes. User chose region when creates a bucket. (https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-creating-buckets)